static/styles/app_components.scss

```diff
@@ -388,6 +388,14 @@
     }
 }
 
+@media (min-width: 750px) and (max-width: 1033px) {
+    .new-style {
+        .stream_sorter_toggle {
+            margin-left: unset;
+        }
+    }
+}
```

Comment:
It might be better to move this to `subscriptions.scss` next to the related rule?  I'm not sure why you have it here.

Reply:
the main style is present in this file 
see this: https://github.com/zulip/zulip/blob/3ceb9fdec2c9a8767877dce3672ae365e7ea338c/static/styles/app_components.scss#L261